src/app/routes/account-gate.tsx

```diff
@@ -1,24 +1,25 @@
 import { ReactNode } from 'react';
```

Comment:
we probably should add a test for this file to make sure nothing breaks these conditions.